buildSrc/src/main/groovy/org/apache/geode/gradle/plugins/DependencyConstraints.groovy

```diff
@@ -249,7 +249,7 @@ class DependencyConstraints implements Plugin<Project> {
       entry('spring-security-oauth2-jose')
     }
 
-    dependencySet(group: 'org.springframework', version: '5.2.5.RELEASE') {
+    dependencySet(group: 'org.springframework', version: '5.2.7.RELEASE') {
```

Comment:
any reason not to bump springframework.security to 5.3.3 at the same time?

Reply:
Oversight perhaps. Will go through this list again when we re-open this PR